### ISSUE TITLE
Add proper avenir font to app

### ISSUE THREAD

**Issue by sunyatax** [juntofoundation/junto-mobile#862]

**Comment by sunyatax** [comment]
Looks like we need to spend $5k+ / year to use Avenir in our mobile app...

Have you run into any circumstances where projects you've worked for had to pay a large sum for commercial typeface licenses or have they deferred to using mostly free fonts? @orestesgaolin

**Comment by orestesgaolin** [comment]
I've worked in a project where we had font designed for our needs and in another project we had font from really nice typeface designers:

- http://velvetyne.fr/fonts/minipax/
- https://rsms.me/inter/

**Comment by Nash0x7E2** [comment]
@sunyatax We can also look at services like Google Fonts to find alternatives. They have a really large collection of fonts

**Comment by orestesgaolin** [comment]
Yup, I'd go with either some libre font or check out some alternatives http://www.identifont.com/find?similar=avenir&q=Go
